src/main/java/com/example/librarymanagement/controller/BookController.java

```diff
@@ -0,0 +1,68 @@
+package com.example.librarymanagement.controller;
+
+import com.example.librarymanagement.dto.BookRequestDto;
+import com.example.librarymanagement.dto.BookResponseDto;
+import com.example.librarymanagement.dto.mapper.BookMapper;
+import com.example.librarymanagement.service.BookService;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/book")
+public class BookController {
+    private final BookService bookService;
+    private final BookMapper bookMapper;
+
+    public BookController(BookService bookService, BookMapper bookMapper) {
```

Comment:
since you're using Lombock you could use @RequiredArgsConstructor annotation on the class and could skip constructor declaration